pynance/textimporter.py

```diff
@@ -0,0 +1,205 @@
+import csv
+import datetime
+
+import pandas as pd
+import numpy as np
+
+
+class CsvImporter():
+    """
+    Importer class to read csv files to well defined pandas.DataFrames
```

Comment:
how about this? We can still construct a class in case we need it - but this is analogous to the `pandas.read_csv` and is easily readable. Any other import could work the same way...
Also the module name also gives context, so `textimporter.CsvImporter.read` is actually redundant, because read and import mean the same thing here

Reply:
I like functions :)!